aspnetcore/fundamentals/http-logging/index.md

```diff
@@ -79,19 +84,19 @@ To configure the HTTP logging middleware, call `AddHttpLogging` in `ConfigureSer
 
 `RequestHeaders` are a set of HTTP Request Headers that are allowed to be logged. Header values are only logged for header names that are in this collection.
 
-[!code-csharp[](samples/6.x/Startup.cs?name=configureservices&highlight=7)]
+[!code-csharp[](samples/6.x/Program.cs?name=Addservices&highlight=5)]
 
 ### `ResponseHeaders`
 
 `ResponseHeaders` are a set of HTTP Response Headers that are allowed to be logged. Header values are only logged for header names that are in this collection.
 
-[!code-csharp[](samples/6.x/Startup.cs?name=configureservices&highlight=8)]
+[!code-csharp[](samples/6.x/Program.cs?name=Addservices&highlight=6)]
```

Comment:
The first time you show `builder.Services.AddHttpLogging`, show the entire file. The following times just the snippet you're now using.

Reply:
done in line number 53